### PR TITLE
feat: Add new local and session storage commands

### PR DESCRIPTION
**Pull request by chrisbreiding**

### CODE REVIEW

File: app/commands/storage.html [hunk 1]
```diff
@@ -62,8 +62,8 @@
 
   <div class="banner">
     <div class="container">
-      <h1>Local Storage</h1>
-      <p>Examples of managing local storage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
+      <h1>Storage</h1>
+      <p>Examples of managing local and session storage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
```
Comment (emilyrohrbough) [inline app/commands/storage.html:66]:
```suggestion
      <p>Examples of managing localStorage and sessionStorage in Cypress, for a full reference of commands, go to <a href="https://on.cypress.io/api" target="_blank">docs.cypress.io</a>
```

File: cypress/e2e/2-advanced-examples/storage.cy.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
+/// <reference types="cypress" />
+
+context('Local / Session Storage', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:8080/commands/storage')
+  })
+  // Although local storage is automatically cleared
```
Comment (emilyrohrbough) [inline cypress/e2e/2-advanced-examples/storage.cy.js:7]: Thinking throughout we should use `localStorage` and `sessionStorage` to match how these are stored on window

Comment (chrisbreiding) [reply cypress/e2e/2-advanced-examples/storage.cy.js]: Addressed in https://github.com/cypress-io/cypress-example-kitchensink/pull/579/commits/7e58c1d53861c2719a6cb2ae191e526ab990723c

File: app/commands/storage.html (hunk 1 above)
Comment (chrisbreiding) [reply app/commands/storage.html]: Addressed in https://github.com/cypress-io/cypress-example-kitchensink/pull/579/commits/7e58c1d53861c2719a6cb2ae191e526ab990723c

File: app/commands/storage.html [hunk 3]
```diff
@@ -87,7 +87,7 @@ <h4 id="clearLocalStorage"><a href="https://on.cypress.io/clearlocalstorage">cy.
   expect(ls.getItem('prop3')).to.be.null
 })
 
-// Clear key matching string in Local Storage
+// Clear key matching string in local storage
```
[the remark this answers is not in the document]
Comment (chrisbreiding) [reply app/commands/storage.html]: Addressed in https://github.com/cypress-io/cypress-example-kitchensink/pull/579/commits/7e58c1d53861c2719a6cb2ae191e526ab990723c

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**